core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/ProjectClasspathChangedListener.java

```diff
@@ -142,7 +142,7 @@ private void visit(IJavaElementDelta delta) {
 	private void notifyClasspathChanged(IJavaProject project) {
 		// refresh catalog if needed
 		IProject prj = project.getProject();
-		String camelVersion = new CamelMavenUtils().getCamelVersionFromMaven(prj);
+		String camelVersion = new CamelMavenUtils().getCamelVersionFromMaven(prj, false);
```

Comment:
I am wondering if it would make sense to use the "false" param as the default if not specifying it.

Reply:
created https://issues.jboss.org/browse/FUSETOOLS-3253